### ISSUE TITLE
".\bootstrap-vcpkg.bat" doesn't work. ERROR: Windows API function CreateFileW got error 0x00000003

### ISSUE THREAD

**Issue by Ghalif** [microsoft/vcpkg#25231]
Hello. I am getting this Error when I used the command "./bootstrap-vcpkg.bat". I followed the instructions according to the link below to build OpenSfM pipeline for photogrammetry.
https://opensfm.org/docs/building.html#installing-dependencies-on-windows

My internet connection is quite stable and I have tried to run the command on different WiFi connections as well. But I get the same error as below.
How can I resolve this? Thank you.

```
PS C:\Windows\system32\OpenSfM\vcpkg> ./bootstrap-vcpkg.bat
Downloading https://github.com/microsoft/vcpkg-tool/releases/download/2022-05-05/vcpkg.exe -> C:\Windows\System32\OpenSfM\vcpkg\vcpkg.exeWhile calling Windows API function CreateFileW got error 0x00000003:
The system cannot find the path specified.

C:\Windows\System32\OpenSfM\vcpkg\scripts\bootstrap.ps1 : Downloading vcpkg.exe failed. Please check your internet
connection, or consider downloading a recent vcpkg.exe from https://github.com/microsoft/vcpkg-tool with a browser.
At line:1 char:4
+ & {& 'C:\Windows\System32\OpenSfM\vcpkg\scripts\bootstrap.ps1' }
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,bootstrap.ps1

ScriptHalted
At C:\Windows\System32\OpenSfM\vcpkg\scripts\bootstrap.ps1:60 char:5
+     throw
+     ~~~~~
    + CategoryInfo          : OperationStopped: (:) [], RuntimeException
    + FullyQualifiedErrorId : ScriptHalted
```

**Comment by autoantwort** [comment]
`C:\Windows\system32` This does not seems to be a good place to install vcpkg.

**Comment by KnowledgeCenter123** [comment]
Yes, `C:\Windows\system32`, I used `C:\src` for vcpkg, it works perfectly.

**Comment by JackBoosY** [comment]
Please do not clone vcpkg in the Windows directory as you do not have permission to perform operations there.

**Comment by Ghalif** [comment]
Its working now. Thank you.

**Comment by canberkirimagzi** [comment]
Hello. I am getting an Error when I use the next command "vcpkg install gmp --triplet=x64-windows". I followed the instructions https://github.com/CGAL/cgal/wiki/Building-GMP-and-MPFR-on-Windows-with-Visual-Studio

My internet connection is quite stable and I have tried to run the command on different WiFi connections as well. But I get the same error as below.
How can I resolve this? Thank you.

warning: The vcpkg C:\Program Files\Microsoft Visual Studio\2022\Community\VC\vcpkg\vcpkg.exe is using detected vcpkg root C:\src\vcpkg and ignoring mismatched VCPKG_ROOT environment value C:\Program Files\Microsoft Visual Studio\2022\Community\VC\vcpkg. To suppress this message, unset the environment variable or use the --vcpkg-root command line switch.
error: Could not locate a manifest (vcpkg.json) above the current working directory.
This vcpkg distribution does not have a classic mode instance.
